TeamCharles/customer-service
# 24 show single incident

## Status
**READY**

## Testing

[N/A ] I have added passing tests for all new functionality.

[X] I have confirmed that all test prior to this pull request still PASS.

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
Shows the incident report for a customer's incident.  If no resolution it prompts for a resolution and saves it to the database.

## Resolves Issue Number
#24

## Deploy Notes


```sh
git pull --prune
git checkout 24-show-single-incident
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

*  Run in Visual Studio
*  You should be prompted to add a resolution
*  Add a resolution
*  Check that it is in the database
======================
*  Run in Visual Studio again
* You should see an incident report with a resolution for the same customer and incident as above
======================
*  Change customer id in `Program.cs` on line 22 to 3
* Run in Visual Studio
* Should receive a different customer and incident with a resolution

## Impacted Files in Application
List general components of the application that this PR will affect:

* Actions/ShowSingleIncidentAction.cs
* Program.cs  _(However this code will be removed before merge)_
